client/app/models/milestone.js

```diff
@@ -137,15 +137,29 @@ export default class MilestoneModel extends Model {
       displayDate = this.dcpActualstartdate;
     }
 
-    if (this.dcpMilestone === FILED_EAS_REVIEW) { // FILED_EAS_REVIEW is used here as the generic "EAS" milestone
-      // 2792: if the project starts with P, prefare the display date found in the "PREPARE_FIELD_EAS" milestone
-      if (this.project.get('dcpName').startsWith('P')) {
-        const prepareFiledEas = this.project.get('milestones').findBy('dcpMilestone', PREPARE_FILED_EAS);
-        if (prepareFiledEas) {
-          displayDate = prepareFiledEas.dcpActualenddate;
+    if ([ //  Task 2792 - "Environmental Assessment Statement Filed" is displayed
+      PREPARE_FILED_EAS,
+      FILED_EAS_REVIEW,
+    ].includes(this.dcpMilestone)) {
```

Comment:
Hmm... why don't we just have two big if blocks to begin with, then? one for PREPARE_FILED_EAS, one for FILED_EAS_REVIEW?